website/client/src/components/groups/membersModal.vue

```diff
@@ -478,20 +478,12 @@ export default {
       let sortedMembers = this.members;
 
       if (!isEmpty(this.sortOption)) {
-        // Use the memberlist filtered by searchTerm
-        if (this.sortOption.value === 'profile.name') {
-          // If members are to be sorted by name, use localeCompare for case-
-          // insensitive sort
-          sortedMembers.sort(
-            (a, b) => a.profile.name.localeCompare(b.profile.name),
```

Comment:
localeCompare or a similar locale-aware case folding technique is still desirable for this so that names in all languages are handled correctly.

Reply:
Sorry ! My bad. Case folding rhymed with case insensitive in my head.